koji-builder/bin/koji-setup.sh

```diff
@@ -55,5 +55,8 @@ check_result 'btype already exists' \
 check_result "user already exists: kojiosbs" \
              koji add-user kojiosbs
 
-check_result 'User already has access to content generator atomic-reactor' \
+check_result 'kojiosbs user already has access to content generator atomic-reactor' \
              koji grant_cg_access kojiosbs atomic-reactor
+
+check_result 'kojiadmin user already has access to content generator atomic-reactor' \
```

Comment:
The error messages must match what koji gives you. If koji was changed to say `{username} user`, then this is fine, otherwise you should keep the original string.

Reply:
ACK